include/deal.II/matrix_free/matrix_free.templates.h

```diff
@@ -1291,10 +1288,12 @@ namespace internal
 
         for (unsigned int no = 0; no < n_dof_handlers; ++no)
           {
+            const DoFHandler<dim> *dofh = &*dof_handler[no];
```

Comment:
```suggestion
            const auto dofh = dof_handler[no];
```
?

Reply:
I made a change: I still spell out `DoFHandler<dim>`, but I got rid of the weird pointer dereferencing (`&*`) by using a reference instead.